app/i18n/en/conf.php

```diff
@@ -33,8 +33,9 @@
 	'display' => array(
 		'_' => 'Display',
 		'darkMode' => array(
-			'_' => 'Automatic dark mode (beta)',
+			'_' => 'Automatic dark mode',
 			'auto' => 'Auto',
+			'help' => 'For selected themes only',
```

Comment:
Would be "For subset themes only" better? There is always a theme selected (!) above.
(I mean that it would be better to have another word than "selected")

Reply:
*some* would also be fine for me

Reply:
Or specific.

Reply:
*For compatible themes only*
Better?

Reply:
Works for me. 👍